src/reducers/profile-view.js

```diff
@@ -721,7 +729,12 @@ export const selectorsForThread = (
       getCallNodeInfo,
       UrlState.getImplementationFilter,
       UrlState.getInvertCallstack,
-      CallTree.getCallTree
+      (...args) => {
+        const callTree = CallTree.getCallTree(...args);
+        // Allow the user access the last viewed call tree via the web console.
+        (window: any).callTree = callTree;
```

Comment:
I wanted to keep this in the reducer, and not have this behavior happen in the `getCallTree` function.

Reply:
So you have it in the selector at the moment, not in the reducer ? As a result I noticed that if we start on a tab that's not the call tree (for example the marker chart), we don't get the `callTree` variable because this selector hasn't been called. But with my suggestion below this would work.